README.md

```diff
@@ -49,6 +49,20 @@ of your `~/.docker/config.json` file to be:
 }
 ```
 
+This will cause `docker` to use `docker-credential-ecr-login` for all remote registries.
+
+To use this helper for a specific ECR registry, use a `credHelpers` section with the name of your ECR registry:
+
+```json
+{
+	"credHelpers": {
+		"aws_account_id.dkr.ecr.region.amazonaws.com": "ecr-login"
+	}
+}
+```
+
+This is useful if you use `docker` to operate on registries that have different authentication credentials.
```

Comment:
This is useful if you use `docker` to operate on registries that use different authentication credentials.